server-side-rendering/.env.example

```diff
@@ -0,0 +1,2 @@
+PUBLIC_APPWRITE_ENDPOINT=
+APPWRITE_KEY=
```

Comment:
Can we prefix it with `PRIVATE`? If not possible thats fine, but it would be cool indicator what is CSR and what is SSR

Reply:
Do we even need it, I didnt find use-case for it in the app. Maybe we don't need whole server-sdk stuff for this example. We might even want to delete user service if we don't use it

Reply:
We need it to ensure secret is return with the createSession and createEmailPasswordSession

Reply:
`PUBLIC_` prefix comes from Vite requirement, I don't think we should use `PRIVATE_` as this implies some vite feature